modules/@apostrophecms/modal/ui/apos/components/AposModal.vue

```diff
@@ -383,14 +383,14 @@ export default {
   }
 
   .apos-modal__main--with-rails {
-    grid-template-columns: 18% 1fr minmax(250px, 22%);
+    grid-template-columns: 18% 1fr minmax(250px, $modal-rail-right-w);
```

Comment:
I have this when visualizing an image:
![image](https://user-images.githubusercontent.com/11479686/93568150-0c963980-f990-11ea-8195-fbe449fac388.png)

The button "published/unpublished" is is wider than the rail (might depend on the window's width). Only the "Save" button is present (and is not doing anything, but that might be done in another PR I guess).

Reply:
That is true. Yes, this stuff would be separate issues. The save button should work in the 3.0 branch right now.